Adyen/Helpers/KeyedDecodingContainerHelpers.swift

```diff
@@ -31,7 +31,7 @@ internal extension KeyedDecodingContainer {
     ///
     /// - parameter key: The key to search for.
     /// - returns: Whether the `Decoder` has an entry for the given key.
-    func containsValue(_ key: KeyedDecodingContainer<K>.Key) throws -> Bool {
+    func containsValue(_ key: KeyedDecodingContainer<K>.Key) throws -> Bool { // swiftlint:disable:this explicit_acl
```

Comment:
I think you can do like [in my PR](https://github.com/Adyen/adyen-ios/pull/856/files#diff-9728e240261c4f07a878219a1fdabce86f7961bf71f4fd2ffac7e468ec0adc64)